app/js/components/searchHistory/searchHistoryComponent.jsx

```diff
@@ -136,6 +137,24 @@ class SearchHistoryComponent extends Component {
     };
   }
 
+/**
+ * checks if there is a saved search history and displays the clear 
+ * search button
+ * @return {void} a button element
+ * @memberof SearchHistoryComponent
+ */
+  displayClearSearchButton() {
```

Comment:
On this function, please consider creating a variable that contains the jsx, then passing that variable to the function, to separate concerns. It also makes the code look cleaner, in my opinion

Reply:
I understand your point and I actually thought of that but that would make more sense if the method is going to be reused. In this case, since I'm checking the session storage first before returning the JSX, I think it will be an overkill doing that.